rero_ils/modules/documents/dumpers/indexer.py

```diff
@@ -254,9 +258,8 @@ def _process_files(self, record, data):
                 if collections:
                     files[file_name]['collections'] = collections
                 if library_pid:
-                    files[file_name]['organisation'] = {
-                        'library_pid': library_pid
-                    }
+                    files[file_name]['library_pid'] = library_pid
+                    files[file_name]['organisation_pid'] = organisation_pid
```

Comment:
By using filename only, it is possible that if you have 2 files with the same name, the metadata will be overwritten.